test/data/h2o2-float-geometry-tran.dat

```diff
@@ -1,7 +1,7 @@
 AR                 0   136.500     3.330     0.000     0.000     0.000
-H                  0.00   145.000     2.050     0.000     0.000     0.000
-H2                 1.00    38.000     2.920     0.000     0.790   280.000
-H2O                2.00   572.400     2.605     1.844     0.000     4.000
+H                  0   145.000     2.050     0.000     0.000     0.000
+H2                 0.999999999999999999    38.000     2.920     0.000     0.790   280.000
+H2O                1.999999999999999999   572.400     2.605     1.844     0.000     4.000
```

Comment:
I see you adjusted these so that we're testing the new behavior, but I think the tests are still going to fail because you need to adjust the code over here: https://github.com/Cantera/cantera/blob/332bb9b9d970c6ead5160f428ff49976fb2947fa/test/python/test_convert.py#L340

Reply:
@bryanwweber what changes would need to be made? Wouldn't the existing test case fail since it would raise an `InputError` with the phrase "geometry flag" in it since `--permissive` won't apply?

Reply:
We also need a test where --permissive is passed and the conversion is successful. There should also be a case testing passing --permissive and the check fails because of the rounding issue.

The existing test is only checking one of the code paths, as you note the one without --permissive

Reply:
@cory-kinney ... I think the only remaining task preventing this from being approved/merged is the test @bryanwweber mentioned?

Reply:
@ischoegl That's correct, I've just been a bit busy so I haven't gotten around to it! I'll try to do that sometime this next week.